dali/test/python/test_operator_random_object_bbox.py

```diff
@@ -0,0 +1,404 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nvidia.dali as dali
+import nvidia.dali.fn as fn
+import nvidia.dali.ops as ops
+import numpy as np
+import scipy.ndimage
+import scipy.ndimage.measurements
+from test_utils import check_batch
+from test_utils import np_type_to_dali
+from nose.tools import raises, assert_raises, nottest
+
```

Comment:
I would maybe consider adding negative tests, stuff like not matching number of weights and classes passed should raise, etc. and if the ignore_classes is specified other class-related arguments should cause an error.

Reply:
I can try.